maven_iuvs/echelle.py

```diff
@@ -2005,7 +2196,7 @@ def writeout_l1c(light_l1a_path, dark_l1a_path, l1c_savepath, light_fits, fit_pa
                                 stdout=outputfile, stderr=errorfile, 
                                 text=True, bufsize=1)
         proc.stdin.write(".com write_l1c_file_from_python.pro\n")
-        time.sleep(3) # Be sure it's compiled 
+        time.sleep(5) # Be sure it's compiled 
```

Comment:
should probably read the output from the terminal to ensure the process has ended, but this can be a todo

Reply:
It has to be read from stdout or stderr or something and it was very hard to get this all working at all, so definitely a todo item